modules/tinymce/src/core/main/ts/api/EditorManager.ts

```diff
@@ -490,10 +490,19 @@ const EditorManager: EditorManager = {
     const initEditors = function () {
       let initCount = 0;
       const editors = [];
-      let targets: Element[];
+      let targets: HTMLElement[];
+
+      const createEditor = function (id: string, settings: RawEditorSettings, targetElm: HTMLElement) {
+        const targetElmWrap = SugarElement.fromDom(targetElm);
```

Comment:
This logic won't work if someone manually creates an editor (which I've seen Joomla and others do). As such it probably needs to be in the `Editor` itself, not in the `EditorManager`.